src/utils/image/index.ts

```diff
@@ -12,21 +12,49 @@ const compressAndPrepareFormData = async (imagePath: string) => {
 };
 
 const uploadImage = async (imagePath: string) => {
-  const formData = await compressAndPrepareFormData(imagePath);
+  try {
+    const formData = await compressAndPrepareFormData(imagePath);
+
+    const imageUrl = await uploadPhoto(formData);
+    return imageUrl;
+  } catch (error) {
+    console.log(error);
+    return imagePath;
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling in the `uploadImage` function could be improved. Currently, it just logs the error and returns the original image path when an error occurs. This might lead to confusion if the caller expects a URL of the uploaded image but gets the local file path instead. Consider throwing the error or returning a more specific error message.

```diff
-    console.log(error);
-    return imagePath;
+    throw new Error(`Failed to upload image: ${error.message}`);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->